parsers/convert-font/README.md

```diff
@@ -22,9 +20,6 @@ interface parser {
 | parameter | Require    | type      | default    | description                                       |
 | --------- | ---------- | --------- | ---------- | ------------------------------------------------- |
 | `formats`    | optional   | `Array<string>`   | `["woff2", "woff", "otf", "ttf", "eot"]` | the list of formats to convert |
```

Comment:
I don't understand why the "optimize-vector" parser has font related parameters. Is this normal?

Reply:
This is a mistake. Fixed.